pkg/ui/.babelrc

```diff
@@ -1,3 +1,6 @@
 {
-  "presets": ["env", "react"]
+  "presets": ["env", "react"],
+  "plugins": [
+    ["import", { "libraryName": "antd", "libraryDirectory": "es", "style": "css" }] // `style: true` for less
```

Comment:
AntD is incredibly aggressive [with their styles](https://ant.design/docs/react/customize-theme#How-to-avoid-modifying-global-styles). Because of this the background of the page in admin ui has been changed to `#fff` instead of the current `#f6f6f6`.

![white-admin-ui](https://user-images.githubusercontent.com/397448/66329618-cbcb1f00-e8fc-11e9-97be-96ff02296e96.png)
![gray-admin-ui](https://user-images.githubusercontent.com/397448/66329654-ddacc200-e8fc-11e9-8368-231ec759b2ab.png)

This is comes from [AntD's base styles](https://github.com/ant-design/ant-design/blob/65eb713fe897f65bbf187e215c664e5f9b468252/components/style/core/base.less#L79) and can be prevented either by [restricting what is imported from antd](https://github.com/ant-design/babel-plugin-import#style) or perhaps by overriding the color lower in the DOM tree. I think modifying `.page` styles would do the trick.

```
// app/containers/layout/layout.styl
.page {
  background-color $background-color
  position relative
  display flex
  flex-direction column
  // etc, etc
```

Reply:
Similarly I believe several fonts have been changed. "Overview dashboard" in the above screenshots is an example, and the top section of the node list/overview page.